olwebsite-app/package.json

```diff
@@ -54,4 +56,4 @@
       "pre-commit": "lint-staged"
```

Comment:
can you make line 56
```
"pre-commit": "lint-staged && npm test -- --watchAll=false"
```

Reply:
Yeah. Is it running the tests before the commit?

Reply:
*running through*